Formula/joern.rb

```diff
@@ -0,0 +1,55 @@
+class Joern < Formula
+  desc "Open-source code analysis platform based on code property graphs"
+  homepage "https://joern.io/"
+  url "https://github.com/joernio/joern/archive/refs/tags/v1.1.1427.tar.gz"
+  sha256 "ceddd43317e2aeb3b7250f4d28a20b866f61b561a32c67930e1ca0b22ddf548a"
+  license "Apache-2.0"
+
+  depends_on "sbt" => :build
+  depends_on "astgen"
+  depends_on "coreutils"
+  depends_on "openjdk"
+  depends_on "php"
+
+  def install
+    system "sbt", "stage"
+
+    libexec.install "joern-cli/target/universal/stage/.installation_root"
+    libexec.install Dir["joern-cli/target/universal/stage/*"]
+
+    bin.write_exec_script libexec/"joern"
```

Comment:
Are these all scripts there? Because I think we have `write_all_` for that

Reply:
I wasn't able to find a `write_all_` style function for exec scripts (ref: https://rubydoc.brew.sh/Pathname.html), but I was able to reduce the many `bin.write_exec_script` calls to:

```rb
bin.write_exec_script Dir[libexec/"*"].select { |f| File.executable? f }
```

which seems to be a solution employed by other existing formula.